update the databricks instructions to be more clear (#32)
rapidsai/spark-examples: docs/databricks.md

  * Select the 3 XGBoost Spark jars and upload them. Save the file names and locations for the step below.

* Add a Startup Init script  
In a terminal create an init script that has commands to copy the jars you imported above over the existing Databricks provided XGBoost jars. Note you will have to change the jar names below to the ones you uploaded.  If you didn't save the names go to the Databricks file browser and look in /FileStore/jars/. If you are using a runtime other then Databricks 5.3 ML with GPU or 5.4 ML with GPU you will have to check the versions of the databricks provided jars in /databricks/jars and update the script accordingly.

```
$ cat /dbfs/databricks/scripts/init.sh
sudo cp /dbfs/FileStore/jars/[dbfs uploaded xgboost4j 0_90.1_Beta jar] /databricks/jars/spark--maven-trees--ml--xgboost--ml.dmlc--xgboost4j--ml.dmlc__xgboost4j__0.81.jar
sudo cp /dbfs/FileStore/jars/[dbfs uploaded cudf_0_8 jar] /databricks/jars/

> sudo cp /dbfs/FileStore/jars/[dbfs uploaded gboost4j_spark 0_90.1_Beta jar] /databricks/jars/spark--maven-trees--ml--xgboost--ml.dmlc--xgboost4j-spark--ml.dmlc__xgboost4j-spark__0.81.jar

Reviewer: ```suggestion
sudo cp /dbfs/FileStore/jars/[dbfs uploaded xgboost4j_spark 0_90.1_Beta jar] /databricks/jars/spark--maven-trees--ml--xgboost--ml.dmlc--xgboost4j-spark--ml.dmlc__xgboost4j-spark__0.81.jar
```

Upload the init.sh script into /databricks/scripts/init.sh.  See https://docs.databricks.com/user-guide/clusters/init-scripts.html for more details about configuring cluster-scoped init script.
```

Start A Databricks Cluster
--------------------------
1. Create a Databricks cluster that meets the above prerequisites. Make sure to use one of the 5.3 ML with GPU or 5.4 ML with GPU Databricks runtimes.
2. Disable autoscaling.
3. Choose the number of workers that matches the number of GPUs you want to use.

> 4. Select a worker type that has 1 GPU for the worker. Such as p3.xlarge.

Reviewer: ```suggestion
4. Select a worker type that has 1 GPU for the worker like p3.xlarge, for example.
```
